### DISCUSSION TITLE
Feat: Support Ghostty terminal

### DISCUSSION THREAD

**Discussion by injust** [aws/codewhisperer-command-line-discussions#193] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
[Ghostty](https://mitchellh.com/ghostty) is a terminal emulator that is currently in private beta. 

You should be able to get access by joining https://discord.gg/ghostty and DM-ing `mitchellh`.

**Comment by grant0417** [comment]
It is hard for us to support terminals that are not open source, but we are open to supporting it in the future.

**Reply by injust** [reply]
It's open source to beta testers currently and it'll be fully open sourced down the road. ~~I'll ping this discussion again when that happens.~~

Edit: I no longer use Amazon Q, so someone else will have to follow up on Ghostty support.

**Reply by sjmillius** [reply]
What is actually needed to support a new terminal technically?
Is this open to open source contributions?

**Reply by newkind** [reply]
@grant0417 ghostty just got a public release and published the source code in their repo - https://github.com/ghostty-org/ghostty

**Reply by cruzluna** [reply]
Would definitely like to see this integration.

**Reply by vinceglb** [reply]
Here is the Github issue for this feature request: https://github.com/aws/amazon-q-developer-cli/issues/323
